src/context/authentication/firebase-dao.ts

```diff
@@ -46,9 +46,9 @@ const authenticationFirebaseDao = (
       .catch((e): Err<Error, void> => err(e, undefined));
   };
 
-  const get = (): Result<Error, string> => {
+  const get = (): Result<Error, Object> => {
     if (user) {
-      return ok(user.uid);
+      return ok({ uid: user.uid, emailVerified: user.emailVerified });
```

Comment:
You can use `const get = (): Result<Error, AuthenticationUser> => {` 
And then return `return ok({ uid: user.uid, emailVerified: user.emailVerified }: AuthenticationUser);`

This can be replicated across anywhere where object is being used :) just helps guarantee we 

- Have the fields we need being populated
- Don't end up with weird data that we don't need